back-end/API/app/controllers/user_controller.py

```
@@ -68,11 +64,15 @@ def register():
         logger.warning('Invalid JSON data format in /register : ' + str(data))
         return jsonify({'message': 'Invalid JSON data format'}), 400
 
-    if user_service.getUser(data['email']) == "<Response 29 bytes [200 OK]>" or user_service.getUser(data['email']) is not None:
-        logger.warning('User already exists')
+    if user_service.getUser(data['email']) is not None:
+        logger.warning(f"User already exists: {data['email']}")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)